third_party/VulkanTutorial/OffscreenRenderingVulkanTutorial.cpp

```diff
@@ -230,22 +230,13 @@ void OffscreenRenderingVulkanTutorial::CreateOffscreenImage() {
   };
   image_extent_ = create_info.extent;
 
-  for (int format_int = VK_FORMAT_BEGIN_RANGE; format_int <= VK_FORMAT_END_RANGE; ++format_int) {
-    const auto format = static_cast<VkFormat>(format_int);
-
-    VkImageFormatProperties format_properties{};
-    VkResult result = vkGetPhysicalDeviceImageFormatProperties(
-        physical_device_, format, create_info.imageType, create_info.tiling, create_info.usage,
-        create_info.flags, &format_properties);
-    if (result == VK_SUCCESS) {
-      image_format_ = format;
-      break;
-    }
-  }
-
-  ORBIT_CHECK(image_format_ != VkFormat::VK_FORMAT_UNDEFINED);
+  VkImageFormatProperties format_properties{};
+  VkResult result = vkGetPhysicalDeviceImageFormatProperties(
+      physical_device_, create_info.format, create_info.imageType, create_info.tiling, create_info.usage,
+      create_info.flags, &format_properties);
+  ORBIT_CHECK(result == VK_SUCCESS);
   ORBIT_LOG("image_format_=%d", image_format_);
-  create_info.format = image_format_;
+
```

Comment:
This code needed to be changed because the BEGIN_RANGE/END_RANGE macros don't exit anymore in newer versions of the vulkan header. So there is no way to iterate over all the VK_FORMATs. I looked at other code (mainly the Vulkan examples from Sascha Willems) to craft this version. It seems to work.